prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -286,10 +286,16 @@ postsubmits:
         - name: github
           mountPath: /etc/github-token
           readOnly: true
+        - name: ssh
+          mountPath: /root/.ssh
       volumes:
       - name: github
         secret:
           secretName: oauth-token
+      - name: ssh
+        secret:
+          secretName: k8s-ci-robot-ssh-keys
```

Comment:
This secret doesn't (and shouldn't) exist on this Prow instance. If an ssh key is needed then we need to use one associated with the bot account for this Prow instance, not k8s-prow.

Reply:
Fixed this so these should be google-oss-robot keys now. I also made changes to the testgrid config to also point to google-oss-robot instead of k8s here: https://github.com/GoogleCloudPlatform/testgrid/pull/324/files, and updated the image to the latest one that has git in it to fix the git error.